_opcodes/op239.html

```diff
@@ -10,7 +10,15 @@
 iwd2: 0
 pst: 0
 ---
-Allows viewing of the current area, based on the 'Type' field.
-<br/>Known values for 'Type' are:
-<br/>0   View explored sections only
-<br/>1   View explored section and unexplored sections.
+Allows viewing of the current area, based on the <code>Type</code> field.
+<br/>Known values for <code>Type</code> are:
+<ul>
+	<li>0 &longrightarrow; View explored sections only</li>
+	<li>1 &longrightarrow; View explored section and unexplored sections</li>
+</ul>
+
+{% capture note %}
+Effect needs to target a creature. Opcode still non-functional in SoD v2.5.
```

Comment:
> Opcode still non-functional in SoD v2.5.

Are you saying they broke the farsight effect and spell and haven't fixed it yet?

Reply:
> > Opcode still non-functional in SoD v2.5.
> 
> Are you saying they broke the farsight effect and spell and haven't fixed it yet?

Yes, it might be fixed in v2.6 (will update as soon as I have a chance to test it...)